src/modelfree/common/utils.py

```diff
@@ -156,6 +171,104 @@ def make_session(graph=None):
     return sess
 
 
+class TrajectoryRecorder(VecMultiWrapper):
```

Comment:
Not hard, you can call `env.render(mode='rgb_array')` to get a bitmap image. I'd be concerned this is overlapping with `VideoRecorder` so it might be good to either find a way to line the two up (e.g. with timestamps) or merge the two classes. Feel free to ping me if you want to discuss this further.